node-runner-cli/setup/SystemD.py

```diff
@@ -84,7 +84,7 @@ def backup_file(filepath, filename, backup_time):
     def set_environment_variables(keystore_password, node_secrets_dir):
         command = f"""
         cat > {node_secrets_dir}/environment << EOF
-        JAVA_OPTS="-server -Xms8g -Xmx8g -XX:+HeapDumpOnOutOfMemoryError -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector"
+        JAVA_OPTS="--enable-preview -server -Xms8g -Xmx8g  -XX:MaxDirectMemorySize=2048m -XX:+HeapDumpOnOutOfMemoryError -XX:+UseCompressedOops -Djavax.net.ssl.trustStore=/etc/ssl/certs/java/cacerts -Djavax.net.ssl.trustStoreType=jks -Djava.security.egd=file:/dev/urandom -DLog4jContextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector"
```

Comment:
Were all these changes intended? What is the intended purpose of `-XX:+UseCompressedOops` and `-XX:MaxDirectMemorySize=2048m`?

Reply:
Both of these changes were missing from previous release. 

 -XX:MaxDirectMemorySize=2048m is used to set the size of the memory that is outside JVM for the process to use. Issue with missed proposals and memory leak was fixed, but this configuration is nice to have to limit the usage of the memory outside JVM 
  -XX:+UseCompressedOops is to reduce jVM memory usage. This setting was tested again during memory leak investigation on missed proposals.  It was missing on CLI as well